packages/@webex/internal-plugin-scheduler/src/payloadTransformer/transformers.js

```diff
@@ -12,54 +12,181 @@ const exampleGeneralTransformer = {
    *
    * @type {string}
    */
-  name: 'exampleGeneralTransformer',
+  name: 'transformSchedulerData',
 
   /**
    * Direction this transformer should process on. This allows for different
    * directions to be handled differently when sending/receiving data.
    *
    * @type {'inbound' | 'outbound' | undefined}
    */
-  direction: undefined,
+  direction: 'inbound',
 
   /**
    * The main transformation function
    * @param {Record<'webex' | 'transform', any>} ctx - An Object containing a webex instance and transform prop.
    * @param {any} data - Data from the event or request.
    * @returns {Promise<any>} - Data after transformation.
    */
-  fn: (ctx, data) =>
-    ctx
-      .transform('exampleInboundTransformer', ctx, data)
-      .then((transformed) => ctx.webex.internal.encryption.example(transformed)),
+  fn: (ctx, data) => {
+    if (!data) {
+      return Promise.resolve();
+    }
+
+    if (!data.encryptionKeyUrl) {
+      return Promise.resolve();
+    }
+
+    // Decrypt participant properties if meeting object contains participants
+    const decryptedParticipants = data.encryptedParticipants
```

Comment:
The general recommendation is to use the transformers to process inbound mercury events and not use them extensively for just processing request body and response body 

I would recommend to just move this business logic to the scheduler file and make it as a helper function